pkg/reconcilers/base_reconciler.go

```diff
@@ -101,7 +102,11 @@ func (b *BaseReconciler) ReconcileResource(obj, desired common.KubernetesObject,
 
 	// item found successfully
 	if common.IsObjectTaggedToDelete(desired) {
```

Comment:
Notice/review new deletion implementation additions please. They are used in the new logic reconcilers

Reply:
:+1: